builders/integrations/bridges/hyperlane.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Cross-Chain Via Hyperlane
+description: Learn how to bridge assets, plus other ways you can connect your Moonbeam dApp to assets and functions on multiple blockchains using Hyperlane protocol & APIs.
+---
+
+# Hyperlane Protocol
+
+![Hyperlane Moonbeam banner](/images/builders/integrations/bridges/hyperlane/hyperlane-banner.png)
+
+## Introduction {: #introduction }
+[Hyperlane](https://hyperlane.xyz){target=_blank} is a security-modular cross-chain communication protocol for Web3. Hyperlane enables dApp users to interact with any asset or application, on any connected chain, with one click. It supports general asset transfer as well as custom cross-chain messaging.  
+
+Using a method called [sovereign consensus](https://docs.hyperlane.xyz/hyperlane-docs-1/protocol/security/sovereign-consensus){target=_blank}, Hyperlane allows developers to configure the method by which messages are sent and validated across chains. Hyperlane is composed of validators, relayers, and watchtowers. Validators will watch for and confirm cross-chain messages. Relayers spend the gas to send messages across chains. Watchtowers perform checks to ensure that validators are good-faith actors, securing the protocol. Take a look at the tech stack diagram and their [protocol documentation](https://docs.hyperlane.xyz/hyperlane-docs-1/protocol/overview){target=_blank} for more details.
+
+![Hyperlane Technology Stack diagram](/images/builders/integrations/bridges/hyperlane/hyperlane-1.png)
```

Comment:
Is there a better image we could use here? It's just really small on the docs site... I assume not, but figured I would check

Reply:
Unfortunately that's the only image that they've made. It's kind of wide, I agree